macaw-semmc/src/Data/Macaw/SemMC/TH.hs

```diff
@@ -971,14 +986,14 @@ defaultAppEvaluator endianness elt interps = case elt of
 data BoolMapOp = AndOp | OrOp
 
 
-evalBoolMap :: A.Architecture arch
+_evalBoolMap :: A.Architecture arch
             => M.Endianness
             -> BoundVarInterpretations arch t fs
             -> BoolMapOp
             -> Bool
             -> BooM.BoolMap (S.Expr t)
             -> MacawQ arch t fs BoundExp
-evalBoolMap endianness interps op defVal bmap =
+_evalBoolMap endianness interps op defVal bmap =
   case BooM.viewBoolMap bmap of
     BooM.BoolMapUnit ->     letTH [| G.addExpr (boolBase $(lift defVal)) |]
     BooM.BoolMapDualUnit -> letTH [| G.addExpr (bNotBase $(lift defVal)) |]
```

Comment:
Are these now unused?

Reply:
Yes, now the `BoolMap` in the `ConjPred` case gets turned into a list via `boolMapList` and then interpreted at runtime by either `allPreds` or `joinPreds`.

Reply:
Can we just delete the unused code then?